index.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Lacquer&family=Poppins&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+body {
+  background-image: url('./img/background_image.png');
+}
+
+a,
+.second-section ul {
+  text-decoration: none;
+  list-style: none;
+}
+
+.nav-bar {
+  display: flex;
+  padding: 20px;
+  font-size: 0.8rem;
+}
+
+.nav-bar ul {
+  flex: 1;
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+  font-size: 15px;
+}
+
+.nav-bar header {
+  flex: 3;
+  font-size: 18px;
+  color: #6070ff;
+  font-weight: bold;
+}
+
+.main-section h1 {
+  font-weight: 900;
+  color: #091e42;
+  font-size: 48px;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  margin: 5% 25%;
+}
+
+.main-section > * {
+  padding: 20px;
+}
+
+.main-section h3 {
+  color: #344563;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.second-section ul {
+  display: flex;
+  flex-direction: row;
+  margin: 10px 0;
+}
+
+.second-section ul li {
+  padding: 10px;
+}
+
+.second-section ul li a img:hover {
+  background-color: rgb(182, 182, 233);
+}
+
+.second-section h3 {
+  color: #7f8cff;
+  font-size: 16px;
+}
+
+@media all and (min-width: 375px) {
+  body {
+    font-size: 16px;
+  }
+}
+
+@media all and (max-width: 768px) {
+  .nav-bar ul {
+    display: none;
+  }
+
+  .nav-bar header {
+    text-align: center;
+  }
+
+  .main-section {
+    margin: 5% 5%;
+  }
+}
```

Comment:
- [ ] This attempt is quite impressive however I think that a few things deserve your attention in order to meet project requirements.
  - [ ] Your `font-size`s are quite different from Figma the design
  - [ ] Your section do not have an appropriate `background-image` for the mobile view
  - [ ] The header section is quite different
Please consider this comparative pictures for more idea:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219270708-53530d57-71d9-4446-a209-65544efc19d5.png)|![image](https://user-images.githubusercontent.com/99715640/219270798-2130efbd-d029-4a68-a9a6-7d7bd3f72b61.png)|

**_Kindly make your design look like the Figma_** :+1:

Reply:
Hello @AbayomiOlaoye,

Regarding the background image, one of the learning objectives is this "Use images and backgrounds to enhance the look of the website." Also, on the video introduction about this project the instructor explicitly says to use an image as a background image .png https://www.loom.com/embed/167236d17f104fc18298c5c9888354c9 (min 4:11)